src/app/public/components/start/start.component.html

```diff
@@ -4,66 +4,92 @@
 		<img class="tw-h-[330px]" src="assets/oeb/images/startpage/sun.png" alt="Hands" />
 	</div>
 	<div class="tw-py-[8rem] tw-relative">
-
-		<h1 class="tw-font-normal md:tw-text-[46px] md:tw-leading-[55.2px] tw-text-black tw-text-[30px] tw-leading-[36px]">{{ 'Start.headline1' | translate }}</h1>
-		<h1 class="lg:tw-pl-[10%] tw-font-black md:tw-text-[46px] md:tw-leading-[55.2px] tw-text-purple tw-text-[30px] tw-leading-[36px]">{{ 'Start.headline2' | translate }}</h1>
+		<h1
+			class="tw-font-normal md:tw-text-[46px] md:tw-leading-[55.2px] tw-text-black tw-text-[30px] tw-leading-[36px]"
+		>
+			{{ 'Start.headline1' | translate }}
+		</h1>
+		<h1
+			class="lg:tw-pl-[10%] tw-font-black md:tw-text-[46px] md:tw-leading-[55.2px] tw-text-purple tw-text-[30px] tw-leading-[36px]"
+		>
+			{{ 'Start.headline2' | translate }}
+		</h1>
 		<div class="tw-flex flex-wrap tw-pt-16 tw-pb-12 tw-flex-col md:tw-flex-row tw-relative">
 			<div class="tw-flex tw-align-middle tw-justify-evenly tw-flex-col tw-w-[100%] md:tw-w-[43%] tw-box-border">
-				<p class="tw-font-normal md:tw-text-[24px] md:tw-leading-[33.6px] tw-text-black tw-text-[16px] tw-leading-[22.4px] tw-pb-4">
+				<p
+					class="tw-font-normal md:tw-text-[24px] md:tw-leading-[33.6px] tw-text-black tw-text-[16px] tw-leading-[22.4px] tw-pb-4"
+				>
 					{{ 'Start.text1' | translate }}
 				</p>
 				<div class="!tw-hidden md:!tw-block">
 					<button
-						class="oeb-button tw-bg-red  tw-text-white"
+						class="oeb-button tw-bg-red tw-text-white"
 						type="submit"
 						loading-message="."
 						[innerHtml]="'Start.badgesErstellen' | translate"
-					></button>		
+					></button>
 				</div>
 			</div>
-			<div class="video-container md:tw-w-[52%] tw-h-[370px] tw-border-purple tw-border-solid tw-border-4 tw-m-4 tw-box-border">
+			<div
+				class="video-container md:tw-w-[52%] tw-h-[370px] tw-border-purple tw-border-solid tw-border-4 tw-m-4 tw-box-border"
+			>
 				<iframe
-					src="https://www.youtube.com/embed/i-dZWdcGZL0"
+					src="https://www.youtube.com/embed/H6lBdH75P7g?si=jiMGc0Jh1LDd2PrQ"
```

Comment:
The `?si=...` query parameter seems to be a tracker, so we should remove it from the link